src/bm_sim/tables.cpp

```diff
@@ -14,16 +14,17 @@
  */
 
 /*
- * Antonin Bas (antonin@barefootnetworks.com)
+ *  Antonin Bas (antonin@barefootnetworks.com)
  *
  */
-
 #include <bm/bm_sim/tables.h>
+
 #include <bm/bm_sim/debugger.h>
 #include <bm/bm_sim/logger.h>
```

Comment:
I would understand a re-ordering to be in lexicographic order but I don't understand the extra empty line?